content/en/logs/guide/lambda-logs-collection-troubleshooting-guide.md

```diff
@@ -0,0 +1,118 @@
+
+## Check if logs are reaching the Live Tail
+
+1. Navigate to the [Log Explorer's Live Tail View][3].
+2. In the Search Bar, use a filter to limit the Live Tail View to just the logs coming from your lambda function. Some common search queries are:
+    * By source: the source is often set to `source:aws` or `source:cloudwatch` but you can find other possible sources in the `parse_event_source` function in the [Lambda function][4]. 
+    * By forwarder name: the Lambda function adds a forwardername tag to all the logs it forwards. You can filter on this tag by searching for `forwardername:*` or `forwardername:<function_name>`.
+3. If you see your logs in the Live Tail View but they are not reaching the Log Explorer, this could be due to one of the following reasons:
+    * Logs are too old: we drop logs with a timestamp older than 6 hours.
+    * If your index has any [exclusion filters][5] set up, they may be filtering out your logs.
+
+## Check the Lambda function monitoring tab
+
+From the Lambda Function: 
+
+1. Click the Monitoring tab.
+
+{{< img src="logs/guide/lambda-monitoring-tab.png" alt="Monitoring tab"  style="width:80%;" >}}
+
+2. The monitoring tab displays a series of graphs indicating the following information about your Lambda function: 
+    * invocations
+    * errors
+    * logs
+
+3. If you don't see any data points on the "Invocations" graph, there may be a problem with the triggers you set for your function. See [Manage Your Function Triggers](#manage-your-function-triggers). To get insight into your lambda invocations without using the monitoring tab, see [Viewing Lambda metrics in Datadog](#viewing-lambda-metrics-in-datadog).
+4. If you see data points on the "Error count and success rate" graph, [Check the Lambda function logs](#check-the-lambda-function-logs) to see what error messages are being reported.
+
+## Viewing Lambda metrics in Datadog
+
+If you have enabled AWS Lambda metrics, you can view metrics related to Lambda invocations and errors within Datadog. The following metrics are all tagged with a key value `functionname` tag: 
+
+* `aws.lambda.invocations ` : Count of times the Lambda function has been triggered / invoked
+* `aws.lambda.errors ` : Count of errors that occurred when the function was invoked
+* `aws.lambda.duration ` : Average amount of time (in milliseconds) that it took for the lambda function to finish executing
+* `aws.lambda.duration.maximum` : Maximum amount of time (in milliseconds) that it took for the lambda function to finish executing
+* `aws.lambda.throttles` : Count of invocation attempts that were throttled due to invocation rates exceeding customer limits
+
+For more information on these and other AWS lambda metrics, see [Amazon Lambda Metrics][6].
+
+## Check the Lambda function logs
```

Comment:
What do you think of `Configuration issue` ?

Reply:
Hmm. My reasoning was to present it more as a set of tools for where people can look to check for indicators of possible issues (monitors, logs). Then the sub-sections after that list some possible issues you could find in each of those places.

In this case they're prompted to check for errors in the logs so they know where to go as a first step when gathering information before creating a support ticket. The API key being misconfigured is just one example of something you might spot from the logs.

Reply:
I leave it up to you. It's the idea of do we want people to see sections has potential things to do or to identify issues that they might have.

Both are valid.

Reply:
Sounds good. I'll go ahead and leave it as-is for now but we can change based on feedback if necessary. (The above comments apply to line 20 as well).